examples/counter/tauri/src-tauri/src/lib.rs

```diff
@@ -0,0 +1,115 @@
+use futures::TryStreamExt;
+use lazy_static::lazy_static;
+use shared::{
+    sse::{SseRequest, SseResponse},
+    App, Capabilities, Core, Effect, Event,
+};
+use std::sync::Arc;
+use tauri::Manager;
+
+mod http;
+use http::http;
+
+mod error;
+use error::Error;
+
+mod sse;
+use sse::sse;
+
+#[derive(Debug)]
+enum Task {
+    Event(Event),
+}
+
+lazy_static! {
+    static ref CORE: Arc<Core<Effect, App>> = Arc::new(Core::new::<Capabilities>());
+}
+
+fn handle_task(
+    task: Task,
+    core: &Arc<Core<Effect, App>>,
+    tauri_app: tauri::AppHandle,
+) -> Result<(), Error> {
+    match task {
+        Task::Event(event) => {
+            for effect in core.process_event(event) {
+                process_effect(effect, core, tauri_app.clone())?
+            }
+        }
+    }
+
+    Ok(())
+}
+
+fn process_effect(
+    effect: Effect,
+    core: &Arc<Core<Effect, App>>,
+    tauri_app: tauri::AppHandle,
+) -> Result<(), Error> {
+    match effect {
+        Effect::Render(_) => {
+            let view = core.view();
+            tauri_app.emit_all("render", view)?
+        }
+        Effect::Http(mut request) => {
+            tauri::async_runtime::spawn({
+                let core = core.clone();
+
+                async move {
+                    let response = http(&request.operation).await.unwrap();
```

Comment:
Not a big one, but would be nicer to use `expect` instead of `unwrap` in these places, to aid diagnosing if they do fail for some reason.